jdaviz/configs/cubeviz/plugins/tests/test_tools.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from astropy.nddata import CCDData
+
+
+@pytest.mark.filterwarnings('ignore:No observer defined on WCS')
+def test_spectrum_at_spaxel(cubeviz_helper, spectrum1d_cube):
+    app = cubeviz_helper.app
+    app.add_data(spectrum1d_cube, 'test')
+    app.add_data_to_viewer("spectrum-viewer", "test")
+    app.add_data_to_viewer("flux-viewer", "test")
```

Comment:
Should we do this to reflect how the API is actually used?
```suggestion
    cubeviz_helper.load_data(spectrum1d_cube, data_label='test')
```